models/menu.py

```diff
@@ -39,14 +39,23 @@
 def get_tooltip_data(tooltip):
     return dict(position="bottom", delay="40", tooltip=tooltip)
 
+def get_friend_req_message():
+    requestcount = db(db.friend_requests.to_h == session.user_id).count()
+    if requestcount == 0:
+        return "No New Friend Requests"
+    elif requestcount == 1:
+        return "1 Pending Friend Request"
+    else:
+        return str(requestcount) + "Pending Friend Requests"
+
 if session.user_id:
     response.menu += [
         (I(_class="fa fa-inbox fa-2x tooltipped",
            data=get_tooltip_data("Notifications")),
          False,
          URL('default', 'notifications'), []),
         (I(_class="fa fa-users fa-2x tooltipped",
-           data=get_tooltip_data("Friend requests")),
+           data=get_tooltip_data(get_friend_req_message())),
```

Comment:
Instead of just changing the tooltip, you can change the color of the icon so that user knows to hover on it, else he/she wouldn't hover at all ?